### DISCUSSION TITLE
Incremental Additive

### DISCUSSION THREAD

**Discussion by karlzw** [casact/chainladder-python#422] [Q&A]
What does this class do?

**Comment by jbogaardt** [comment]
https://chainladder-python.readthedocs.io/en/latest/development.html#incrementaladditive

**Reply by karlzw** [reply]
I have read this a couple of times but I still don't get it. Can I use it to adjust for inflation on a triangle if I have the inflation rates?

**Reply by jbogaardt** [reply]
Yes that is a key benefit of this method. The estimator has provision for prospective trend to account for your views on how much calendar year inflation you expect:
Refer to the [class definition](https://chainladder-python.readthedocs.io/en/latest/modules/generated/chainladder.IncrementalAdditive.html).

I believe the inspiration for the estimator is [Section 4.7 of this paper](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.376.544&rep=rep1&type=pdf)
